src/build

```diff
@@ -16,13 +16,22 @@ base_dir=$(cd "`dirname "$loc"`" && pwd)
 
 need_to_build=0
 dependencies=("build.sbt" "project/plugins.sbt" "source")
-launcher="target/start"
+
+# Determine os version and choose correspondent .sh or .bat file accordingly.
+os_version="`uname -a`"
+
 
 if [ $# -ne 0 ]; then
     echo "Error: No command-line arguments allowed.  Got: $@" > /dev/stderr
     exit 1
 fi
 
+if [[ $os_version == *"MINGW"* ]]; then
+    launcher="target/start.bat"
```

Comment:
I vote for this change we don't even worry about making this run in CMD on windows, so revert this whole file and save it for another PR (but since Windows 10 supports running bash on ubuntu I don't think it's even worth pursuing in the future either)